sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -134,9 +135,15 @@ private object ItemWriteStrategy extends Enumeration {
       throw new IllegalArgumentException("name is not a valid ItemWriteStrategy"))
 }
 
-private case class CosmosWriteConfig(itemWriteStrategy: ItemWriteStrategy, maxRetryCount: Int)
+private[spark] case class CosmosWriteConfig(itemWriteStrategy: ItemWriteStrategy, maxRetryCount: Int, bulkEnabled: Boolean)
 
 private object CosmosWriteConfig {
+  private val bulkEnabled = CosmosConfigEntry[Boolean](key = "spark.cosmos.write.bulkEnabled",
```

Comment:
I assume this is temporary, correct? I guess ultimately there is no reason ever to not use bulk?

Reply:
my thought is once bulk is fully ready and tested we should enable bulk by default `spark.cosmos.write.bulkEnabled=true` as default. However I would still like to support a fall back scenario that if for any reason we want to disable bulk to switch back to point operation we can (then the user can explicitly set `spark.cosmos.write.bulkEnabled=false` to disable.